dotcom-rendering/src/components/Dateline.tsx

```diff
@@ -46,20 +47,30 @@ export const Dateline = ({
 	secondaryDateline,
 	format,
 }: Props) => {
-	const styles = [
-		datelineStyles,
-		format.design === ArticleDesign.LiveBlog && standfirstColouring,
-	];
+	const { renderingTarget } = useConfig();
+	const isLiveBlog = format.design === ArticleDesign.LiveBlog;
+	const isApps = renderingTarget === 'Apps';
 
+	const mobileColor = {
+		'--mobile-color': isApps
+			? palette('--dateline-mobile')
+			: isLiveBlog
+			? palette('--standfirst-text')
+			: undefined,
```

Comment:
Would this be a litte safer and more explicit?

```suggestion
			: 'inherit',
```